pkg/config/util.go

```diff
@@ -39,7 +39,7 @@ func makeKubeConfig(clientKey *rsa.PrivateKey, clientCert, caCert *x509.Certific
 			{
 				Name: clustername,
 				Cluster: v1.Cluster{
-					Server: "https://" + endpoint,
+					Server:                   "https://" + endpoint,
```

Comment:
?

Reply:
@jim-minter is this how your gofmt formatted this file? I ask because this is exactly how my gofmt works. It however causes issues for our `make verify` target.

@kargakis  I believe i've discussed this issue with you in the past but we never arrived at a cause/solution.